packages/README.md

```diff
@@ -101,10 +107,17 @@ These files are, by default, only fetched from our upstream source mirror, using
 
 If your source is not yet available in the upstream source mirror, you can run `cargo make` with `-e BUILDSYS_UPSTREAM_SOURCE_FALLBACK=true`.
 
-The [build-dependencies](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#build-dependencies) table is used to declare a dependency on another crate.
+The [build-dependencies](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#build-dependencies) table is used to declare a build-time dependency on another package.
 In this case, `libwoof` depends on `glibc` and `libseccomp`.
 We want those libraries to be built first, and for this one to be rebuilt when they are modified.
 
+We use the [dependencies](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html) section to specify any packages that are needed at runtime (that is, RPM `Requires:`).
+Although we don't need these to be built *before* the current package, we do need them to be built before we construct the Bottlerocket image.
+With build-dependencies (`BuildRequires:`) it's important that they be built *before* the current package is built.
+Runtime dependencies (`Requires`) can be built at any time.
+The separation between `build-dependencies` and `dependencies` in Cargo.toml is not necessary but helps us distinguish between the two dependency types.
+Note that we do not include `Requires:` dependencies if they are already satisfied by either the current package, or the build-dependencies.
```

Comment:
> I think it could be clearer to remove the "# RPM Requires" comments in the Cargo.toml

The closest thing we have to a mapping is: `Requires:` -> `dependencies` and `BuildRequires:` -> `build-dependencies`. The mapping is true even if some are omitted.

I definitely think it's more helpful than not to have the `# RPM Requires` comment in Cargo.toml even if you need the README to fully understand what's going on.

I'll take another stab at explaining it better in the README.